src/TLuaInterpreter.cpp

```diff
@@ -6780,6 +6799,16 @@ int TLuaInterpreter::exists( lua_State * L )
             it1++;
         }
     }
+    //TODO: exists(keybind) doesn't currently work, figure out why
+    else if( type == "keybind")
+    {
+        QMap<QString, TKey *>::const_iterator it1 = pHost->getKeyUnit()->mLookupTable.find( name );
+        while( it1 != pHost->getKeyUnit()->mLookupTable.end() && it1.key() == name )
```

Comment:
They did end up being QMultiMaps, but count() still worked as expected.

It seems that KeyUnit is not using mLookupTable in the same way that AliasUnit does, and since exists() relies on mLookupTable, that's the reason I can't get exists() to work with keybinds.

Couple things yet to try, but I should find out why AU and KU are tracking their children in different ways first, because I don't want to accidentally break something.